### ISSUE TITLE
Strange render on MacOS 10.14 (mojave)

### ISSUE THREAD

**Issue by rageworx** [rageworx/FLSWRender#10]
Strange error rendering on MacOS 10.14 mojave.

<img width="510" alt="image" src="https://user-images.githubusercontent.com/6948225/120512462-592e8180-c406-11eb-9ece-5a20c7ce68db.png">

**Comment by rageworx** [comment]
This strange error occurs on optimized runtime, debuging not happens like this.

**Comment by rageworx** [comment]
Other object test, there's error detected as 
```
Current executable set to 'bin/test' (x86_64).
(lldb) r
Process 3303 launched: '/Users/raphaelkim/Projects/FLSWRender/bin/test' (x86_64)
Process 3303 stopped
* thread #1, queue = 'com.apple.main-thread', stop reason = EXC_BAD_ACCESS (code=1, address=0x10228c000)
    frame #0: 0x0000000100068bc8 test`renderContext::clearBuffer(this=0x0000000100304d30, earsebuff=true) at FLSWRenderer.cpp:590:26
   587 	        #pragma omp parallel for
   588 	        for ( size_t cnt=0; cnt<=cmax; cnt++ )
   589 	        {
-> 590 	            zbuffer[cnt] = 1000000.f; /// was 10000.0f;
   591 	        }
   592 	    }
   593 	}
Target 0: (test) stopped.
```

**Comment by rageworx** [comment]
Fixed bugs commited at 99a42e0..3f50e01
